src/VideoJS.as

```diff
@@ -128,6 +128,10 @@ package{
                 _app.model.preload = String(loaderInfo.parameters.preload);
             }
 
+            if(loaderInfo.parameters.muted != undefined){
+                _app.model.muted = loaderInfo.parameters.muted;
```

Comment:
this basically loads in the `muted` attribute value when set? I can't believe that we this was the issue, lol.